### ISSUE TITLE
[Suggestion] Commands listed in '/listgymcommands' are given a unique ID which '/delgymcommand uses'.

### ISSUE THREAD

**Issue by HackuJacku** [AnDwHaT5-Sidemods/AGP-Responged#32]
Commands listed in '/listgymcommands' are given a unique ID which '/delgymcommand uses'.

Commands in '/listgymcommands' would appear like the following:
[1] Command 1
[2] Command 2
[3] Command 3

The '/delgymcommand' syntax would change to '/delgymcommand <gym> <ID>'.
If I wanted to remove "Command 1", I therefore would do '/delgymcommand <gym> 1'.

**Comment by clienthax** [comment]
Would this work better
a list of clickable commands that can be removed with a confirm option?

**Comment by HackuJacku** [comment]
Yes, that would do fine! Typing out the full command is a pain to do.
